.travis/test.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+if [ $RUN_TESTS = "OFF" ]
```

Comment:
I know there is no problem in our current usage because RUN_TESTS should always be initialized, but just in case:
```suggestion
if [ "$RUN_TESTS" = "OFF" ]
```

The same for the COVERALLS test a few lines below.